.dockerignore

```diff
@@ -3,4 +3,6 @@ bundles
 vendor/pkg
 .go-pkg-cache
 .git
+Dockerfile
+.dockerignore
```

Comment:
Could you perhaps sort this file alphabetically?

Also, looks like there's still entries in the `.gitignore` for the auto-generated things;  https://github.com/moby/moby/blob/42f0a0db75a921145c7f519f7b550e1392890da2/.gitignore#L20-L21


(there's some other old things in there, e.g. the `rpm` contrib, and I don't think `vendor/pkg` is still needed, but we can do that in a follow-up)